tests/cpu/ops/fixedfunction/FixedFunctionOpCPU_tests.cpp

```diff
@@ -405,7 +406,518 @@ OCIO_ADD_TEST(FixedFunctionOpCPU, aces_gamut_map_13)
+
+    GenerateIdentityLut3D(input_32f.data(), lut_size, num_channels, OCIO::LUT3DORDER_FAST_RED);
+
+    OCIO::FixedFunctionOpData::Params params = {
+        // Peak luminance
+        100.f,
+        // Rec709 gamut
+        0.6400, 0.3300, 0.3000, 0.6000, 0.1500, 0.0600, 0.3127, 0.3290
+    };
+
+    OCIO::ConstFixedFunctionOpDataRcPtr funcData
+        = std::make_shared<OCIO::FixedFunctionOpData>(OCIO::FixedFunctionOpData::ACES_OUTPUT_TRANSFORM_20_INV,
+                                                      params);
+
+    OCIO::ConstOpCPURcPtr op;
+    OCIO_CHECK_NO_THROW(op = OCIO::GetFixedFunctionCPURenderer(funcData));
+    OCIO_CHECK_NO_THROW(op->apply(&input_32f[0], &output_32f[0], num_samples));
+
+    OCIO::ConstFixedFunctionOpDataRcPtr funcData2
+        = std::make_shared<OCIO::FixedFunctionOpData>(OCIO::FixedFunctionOpData::ACES_OUTPUT_TRANSFORM_20_FWD,
+                                                      params);
+
+    ApplyFixedFunction(&output_32f[0], &input_32f[0], num_samples,
+                       funcData2,
+                       1e-3f,
+                       __LINE__);
+}
+
+OCIO_ADD_TEST(FixedFunctionOpCPU, aces_ot_20_p3d65_100n_rt)
+{
+    const int lut_size = 8;
+    const int num_channels = 4;
+    int num_samples = lut_size * lut_size * lut_size;
+    std::vector<float> input_32f(num_samples * num_channels, 0.f);
+    std::vector<float> output_32f(num_samples * num_channels, 0.f);
+
+    GenerateIdentityLut3D(input_32f.data(), lut_size, num_channels, OCIO::LUT3DORDER_FAST_RED);
+
+    OCIO::FixedFunctionOpData::Params params = {
+        // Peak luminance
+        100.f,
+        // P3D65 gamut
+        0.680, 0.320, 0.265, 0.690, 0.150, 0.060, 0.3127, 0.3290
+    };
+
+    OCIO::ConstFixedFunctionOpDataRcPtr funcData
+        = std::make_shared<OCIO::FixedFunctionOpData>(OCIO::FixedFunctionOpData::ACES_OUTPUT_TRANSFORM_20_INV,
+                                                      params);
+
+    OCIO::ConstOpCPURcPtr op;
+    OCIO_CHECK_NO_THROW(op = OCIO::GetFixedFunctionCPURenderer(funcData));
+    OCIO_CHECK_NO_THROW(op->apply(&input_32f[0], &output_32f[0], num_samples));
+
+    OCIO::ConstFixedFunctionOpDataRcPtr funcData2
+        = std::make_shared<OCIO::FixedFunctionOpData>(OCIO::FixedFunctionOpData::ACES_OUTPUT_TRANSFORM_20_FWD,
+                                                      params);
+
+    ApplyFixedFunction(&output_32f[0], &input_32f[0], num_samples,
+                       funcData2,
+                       1e-2f,
+                       __LINE__);
+}
+
+OCIO_ADD_TEST(FixedFunctionOpCPU, aces_ot_20_p3d65_1000n_rt)
+{
+    const int lut_size = 8;
+    const int num_channels = 4;
+    int num_samples = lut_size * lut_size * lut_size;
+    std::vector<float> input_32f(num_samples * num_channels, 0.f);
+    std::vector<float> output_32f(num_samples * num_channels, 0.f);
+
+    GenerateIdentityLut3D(input_32f.data(), lut_size, num_channels, OCIO::LUT3DORDER_FAST_RED);
```

Comment:
I actually missed the fact that the test does inv, then fwd, so please ignore my comment about trying to improve the spacing with a gamma. But you are right that perhaps the input should be scaled down so that you don't have the top quarter being clipped and therefore a less useful test point.

Reply:
Actually because this is not a full inversion from PQ, I think it actually needs to be scaled up to 10.0 (which would produce 1000 nits as 1.0 is 100 nits).